drivers/bmx055/bmx055_internal.h

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (C) 2017 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     drivers_bmx055
+ *
+ * @{
+ * @file
+ * @brief       Definitions for the bmx055 device
+ *
+ * @author      Semjon Kerner <semjon.kerner@fu-berlin.de>
+ */
+
+#ifndef BMX055_INTERNAL_H
+#define BMX055_INTERNAL_H
+
+/* MAGNETOMETER */
+
+#define REG_MAG_CHIPID      (0x40U)
```

Comment:
Please add some doxygen documentation in this file (only `@name` blocks are required I think)